test/utils/__renders__/1.html

```diff
@@ -1 +1 @@
-<svg>
\ No newline at end of file
+data:application/json;base64,eyJuYW1lIjogIkVOQ0xPU0VELk1PTkVZIC0gIzY3NzAwOTUzMjEyNDQ4MTM5OTE4Mzk3ODU3NzgwOTgzNDM2NTU4MTEwMTA2NTM4NDkzNzk3MTc1NjkzOTU3OTczNjk5OTUzOTUwNzIxIiwgImRlc2NyaXB0aW9uIjogIlR1cm4gWW91ciBNYWdpYyBJbnRlcm5ldCBNb25leSBJbnRvIGFuIE5GVCAtLSBCZWNhdXNlLCBXaHkgTm90PyEiLCAiaW1hZ2UiOiAiZGF0YTppbWFnZS9zdmcreG1sO2Jhc2U2NCxQSE4yWnlCNGJXeHVjejBpYUhSMGNEb3ZMM2QzZHk1M015NXZjbWN2TWpBd01DOXpkbWNpSUhkcFpIUm9QU0l6TlRBaUlHaGxhV2RvZEQwaU16VXdJaUJ6ZEhsc1pUMGlZbUZqYTJkeWIzVnVaRG9qTURBd08yWnZiblF0Wm1GdGFXeDVPa052ZFhKcFpYSWdUbVYzSWo0OGRHVjRkQ0I0UFNJMU1DVWlJSGs5SWpVd0pTSWdkR1Y0ZEMxaGJtTm9iM0k5SW0xcFpHUnNaU0lnWm1sc2JEMGlJMlptWmlJZ1ptOXVkQzF6YVhwbFBTSTRNQ0lnWm05dWRDMTNaV2xuYUhROUltSnZiR1FpUGpFOEwzUmxlSFErUEhSbGVIUWdlRDBpTlRBbElpQjVQU0kyTkNVaUlIUmxlSFF0WVc1amFHOXlQU0p0YVdSa2JHVWlJR1pwYkd3OUlpTm1abVlpSUdadmJuUXRjMmw2WlQwaU5EQWlQbFZUUkVNOEwzUmxlSFErUEhSbGVIUWdlRDBpTWpBaUlIazlJak14TUNJZ1ptbHNiRDBpSXprNU9TSWdabTl1ZEMxemFYcGxQU0l4TmlJK1ZWTkVJRU52YVc0OEwzUmxlSFErUEhSbGVIUWdlRDBpTWpBaUlIazlJak16TUNJZ1ptbHNiRDBpSXprNU9TSWdabTl1ZEMxemFYcGxQU0l4TWlJK01IZzVOV0ZrTmpGaU1HRXhOVEJrTnpreU1UbGtZMlkyTkdVeFpUWmpZekF4WmpCaU5qUmpOR05sUEM5MFpYaDBQand2YzNablBnPT0ifQ==
```

Comment:
The JS script should try to decode this and extract the SVG

Reply:
done! 👍